Makefile

```diff
@@ -1,17 +1,29 @@
 SHELL=bash
 
 BUILD=build
-BUILD_ARCH=$(BUILD)/$(GOOS)-$(GOARCH)
 BIN_DIR?=.
 
-export GOOS?=$(shell go env GOOS)
-export GOARCH?=$(shell go env GOARCH)
-
 .PHONY: build
 build:
-	@mkdir -p $(BUILD_ARCH)/$(BIN_DIR)
-	go build -o $(BUILD_ARCH)/$(BIN_DIR)/dp-api-poc main.go
+	@mkdir -p $(BUILD)/$(BIN_DIR)
+	go build -o $(BUILD)/$(BIN_DIR)/dp-apipoc-server main.go
```

Comment:
this new name will probably require a change to dp-setup to follow the rename:
cf
- https://github.com/ONSdigital/dp-setup/blob/develop/ansible/files/vault-policies/dp-api-poc.hcl#L1
- and https://github.com/ONSdigital/dp-setup/blob/develop/ansible/templates/consul-template/web-nginx.http.conf.tpl.j2#L306